tests/system/robot/symbolPronunciationTests.py

```diff
@@ -313,25 +309,11 @@ def test_moveByChar():
 }
 
 
-def _wait_for_specific_speech(speech: str, maxWaitSeconds: int) -> bool:
```

Comment:
This has been removed in favour of improving the API of `spy.wait_for_specific_speech`